src/services/react.ts

```diff
@@ -0,0 +1,17 @@
+import MustReloadDialog from 'components/shared/MustReloadDialog';
+import { lazy } from 'react';
+import { logRocketConsole, logRocketEvent } from './shared';
+import { CustomEvents } from './types';
+
+const handledLazy = (factory: () => Promise<{ default: any }>) => {
```

Comment:
I am not the _biggest_ fan of this function name, but it gets the general idea across and I cannot think of a better alternative at the moment.

Reply:
yeah... I had no clue what to use without getting way too verbose.